src/commands/build.rs

```diff
@@ -376,20 +376,24 @@ impl BuildCommand {
             )
         } else {
             match (
-                env::var("CI_REGISTRY").ok(),
-                env::var("CI_PROJECT_NAMESPACE").ok(),
-                env::var("CI_PROJECT_NAME").ok(),
-                env::var("GITHUB_REPOSITORY_OWNER").ok(),
-                self.registry.as_ref(),
-                self.registry_path.as_ref(),
+                env::var("CI_REGISTRY").ok().map(|s| s.to_lowercase()),
+                env::var("CI_PROJECT_NAMESPACE")
+                    .ok()
+                    .map(|s| s.to_lowercase()),
+                env::var("CI_PROJECT_NAME").ok().map(|s| s.to_lowercase()),
+                env::var("GITHUB_REPOSITORY_OWNER")
+                    .ok()
+                    .map(|s| s.to_lowercase()),
+                self.registry.as_ref().map(|s| s.to_lowercase()),
+                self.registry_path.as_ref().map(|s| s.to_lowercase()),
             ) {
                 (_, _, _, _, Some(registry), Some(registry_path)) => {
                     trace!("registry={registry}, registry_path={registry_path}");
                     format!(
                         "{}/{}/{}",
-                        registry.trim().trim_matches('/').to_lowercase(),
-                        registry_path.trim().trim_matches('/').to_lowercase(),
-                        recipe.name.trim().to_lowercase()
+                        registry.trim().trim_matches('/'),
+                        registry_path.trim().trim_matches('/'),
```

Comment:
trim().trim_matches() looks like a code smell lol

Reply:
The first trim removes whitespace, the second removes any `/` characters on the start and end so that we don't accidentally have a double `//` in the image path